eos-tech-support/eos-hack-reset-quest

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+usage () {
+    cat <<EOF
+Usage: $0 QUEST-NAME|all
+Reset the state of QUEST-NAME, you can specify "all",
+to do a full reset of all the quests and a reset of the
+desktop icons generated by quests.
+
+  -h, --help                display this help and exit
+
+EOF
+}
+
+args=$(getopt -o h -l "help" -n "eos-hack-reset-quests" -- "$@")
```

Comment:
Nitpick: this -n is inconsistent with the binary name.

Reply:
($0 will always match!)

Reply:
`$(dirname "$0")` is even better!